### PR TITLE
Minor security enhancements (captcha)

### PR DESCRIPTION
**Pull request by jia1**
schema.rb was changed unintentionally. Please check if this is fatal.

### CODE REVIEW

File: app/assets/javascripts/connect.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 $(document).ready(function() {
+  // dynamically write the email address
+  $(".connect-email").html("conect@nuscomputing.com");
```
Comment (nicholaslum444) [inline app/assets/javascripts/connect.js]: connect spell wrong

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -20,6 +20,10 @@
   <meta name="twitter:title" content="NUS Students' Computing Club" />
   <meta name="twitter:description" content="A Constituent Club of the NUS Students' Union." />
   <meta name="twitter:image" content="<%= image_path "home/highlight-0.jpg" %>" />
+  
+  <!-- SEO metas -->
+  <meta name="description" content="We are the official student club of NUS School of Computing. We plan events for the undergraduate student body and ensure the welfare of the students are taken care of. Fun, games, sports and camps are organised by us to complement the intensive academics that students go through daily." />
+  <meta name="keywords" content="nuscomputing, computing, soc, nus" />
```
Comment (nicholaslum444) [inline app/views/layouts/application.html.erb]: something similar already exists! check line 7, You may update it if you wish, but don't duplicate it

File: app/views/pages/connect.html.erb [hunk 3]
```diff
@@ -16,7 +19,8 @@
   	<ul class="vcard connect-info">
   		<li><i class="fa fa-home">&nbsp;</i>13 Computing Drive, Computing 1, #02-14, Singapore 117417</li>
   		<li><i class="fa fa-phone">&nbsp;</i>+65 6516 6759</li>
-  		<li><i class="fa fa-envelope-o">&nbsp;</i>connect@nuscomputing.com</li>
+  		<!-- go to app/assets/javascripts/connect.js to update the contact email -->
+  		<li><i class="fa fa-envelope-o">&nbsp;</i><span class="connect-email"></span></li>
```
Comment (nicholaslum444) [inline app/views/pages/connect.html.erb]: why does this need to be dynamic? if client disables javascript, then the email won't show up

**Comment by nicholaslum444** [comment]
@jiayeerawr Thanks for the improvements! Gimme a day or two to check the schema. Am abit busy now

**Comment by jia1** [comment]
1. Switch back contact email from dynamic content to static content
2. Remove duplicate meta tags

File: app/views/pages/_form.html.erb [hunk 4]
```diff
@@ -28,6 +31,16 @@
 	    </label>
 		</div>
 	</div>
+	
+	<!-- ReCaptcha -->
+	<div class="row>
```
Comment (kennho) [inline app/views/pages/_form.html.erb]: The div class="row
is missing a closing "

**Comment by jia1** [comment]
Did not do the server side integration, the latest commits are attempts of doing so, however, same problem remains

**Comment by nicholaslum444** [comment]
Closed as it is opened for too long. Will reopen if there are updates

**Pull request closed without merging**